web-app/CKS.Web/Startup.cs

```diff
@@ -42,10 +45,34 @@ public void ConfigureServices(IServiceCollection services)
 			}
 
 			services.AddControllers();
+
+			if (!CurrentEnvironment.IsDevelopment())
+			{
+				services.AddHsts(options =>
+				{
+					//TODO: Investigate why https://hstspreload.org/ times out for nice.org.uk currently
+					options.Preload = true;
+					options.IncludeSubDomains = true;
+					options.MaxAge = TimeSpan.FromDays(60);
+				});
+
+				services.AddHttpsRedirection(options =>
```

Comment:
I don't think we need HTTPS redirection as part of the app, as it's [already handled by the nginx instance](https://github.com/nice-digital/nginx-geoipfilter/blob/master/nginx/geoip/geoip#L6-L8) that sits in front of CKS. Having said that, it probably won't hurt to include it here, it just won't do anything as the x-forwarded-proto header will already be hTTPS by the time the request gets to the app. Including it here should mean that dev/test still redirect to HTTPs but then I'm not sure I like them being too different from beta/live. Maybe one to discuss?